### PR TITLE
Github Actions backports from stabilization branch

### PR DESCRIPTION
**Pull request by evgenyz**
- Rename PR gating workflows (for clarity)
- Introduce release workflow (back-ported from former stabilization)
- Introduce stabilization workflow (back-ported from former stabilization)
- Update the release docs (new automated release process)
- Add scapval 1.3 to the stabilization workflow

### CODE REVIEW

File: release_tools/README.md [hunk 1]
```diff
@@ -102,113 +49,64 @@ Before the release
...
-Requests merged in this release.
-Not all PRs have an entry to avoid over cluttering. To determine the
-*relevant* PRs a few heuristics are applied:
+Everything necessary for the release is built by the release GitHub Action,
+which is triggered when a tag **v*.*.*** is pushed to the repository. This tag
+should point to the **stabilization** branch that is ready to be released.
 
--   If a Profile file (`.profile`) was changed, it will be listed under
-    Profiles section;
+This action will also create a relase draft with release notes automatically
```
Comment (vojtapolasek) [inline release_tools/README.md:71]:
```suggestion
This action will also create a release draft with release notes automatically
```

File: release_tools/README.md [hunk 2]
```diff
@@ -102,113 +49,64 @@ Before the release
...
+generated. They way how action will categorize and break down changes is
```
Comment (vojtapolasek) [inline release_tools/README.md:72]:
```suggestion
generated. The way in which the action will categorize and break down changes is
```

File: release_tools/README.md [hunk 3]
```diff
@@ -102,113 +49,64 @@ Before the release
...
-A file named `release-notes.txt` will contain the notes, **review the
-file** and make changes if necessary.
 
 Release
 -------
 
-After Jenkins builds have finished, release notes were generated and
-reviewed. We move on to creating the release entry in GitHub:
+Create and push **vX.X.XX** tag to the GitHub repo. Wait for the release action
+to finish. Check the release draft and update the release notes if nessesary.
```
Comment (vojtapolasek) [inline release_tools/README.md:82]:
```suggestion
to finish. Check the release draft and update the release notes if necessary.
```

File: release_tools/README.md [hunk 4]
```diff
@@ -66,8 +12,9 @@ bringing a sudden breaking change into the release.
 
 The stabilization phase is typically two weeks long. To start the stabilization:
 
-- Create a new branch called stabilization-vx.y.z, where x.y.z is the version of
-  the upcoming release
+- Create a new branch called **stabilization**, if there is a leftover branch
+  from the previous release the new one shoule replace it, we do not version
```
Comment (vojtapolasek) [inline release_tools/README.md:16]:
```suggestion
  from the previous release the new one should replace it, we do not version
```

File: release_tools/README.md [hunk 5]
```diff
@@ -229,7 +131,8 @@ Fedora builds
 -   and [Package Update How
     To](https://fedoraproject.org/wiki/Package_update_HOWTO)
 
-Copr builds
+
+COPR Builds
```
Comment (vojtapolasek) [inline release_tools/README.md:135]: I wonder if  we should leave the documentation here since we no longer maintain the copr repo for SSG.

**Review by vojtapolasek — CHANGES_REQUESTED** [review]
Thank you for the PR. I have just some typo suggestions in the documentation, the rest looks fine.

File: release_tools/README.md (hunk 5 above)
Comment (evgenyz) [reply release_tools/README.md]: Got rid of it

**Review by vojtapolasek — APPROVED** [review]
Looks good, thank you.

**Pull request merged**